### PR TITLE
[TACHYON-1637] Add the Chinese version of documentation page 'Contributing-to-Tachyon'

### PR DESCRIPTION
**Pull request by saltylin**
https://tachyon.atlassian.net/browse/TACHYON-1637

### CODE REVIEW

File: docs/cn/Contributing-to-Tachyon.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+也可以运行以下命令将M2_REPO添加到classpath变量中：
+
+{% include Contributing-to-Tachyon/M2_REPO.md %}
+
+如果你使用的是IntelliJ IDEA，你可能需要修改Maven配置中的'developer'以防止导入错误，可以通过以下方式进行：
+
+    View > Tool Windows > Maven Projects
+
+### Presentations
```
Comment (ifcharming) [inline docs/cn/Contributing-to-Tachyon.md]: 幻灯片介绍？

File: docs/cn/Contributing-to-Tachyon.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+[SlideShare](http://www.slideshare.net/TachyonNexus/tachyon-presentation-at-ampcamp-6-november-2015)
+-   Strata and Hadoop World 2015 (September, 2015)
+[SlideShare](http://www.slideshare.net/TachyonNexus/tachyon-an-open-source-memorycentric-distributed-storage-system)
+-   Strata and Hadoop World 2014 (October, 2014)
+[pdf](http://www.cs.berkeley.edu/~haoyuan/talks/Tachyon_2014-10-16-Strata.pdf)
+[pptx](http://www.cs.berkeley.edu/~haoyuan/talks/Tachyon_2014-10-16-Strata.pptx)
+-   Spark Summit 2014 (July, 2014) [pdf](http://goo.gl/DKrE4M)
+-   Strata and Hadoop World 2013 (October, 2013) [pdf](http://goo.gl/AHgz0E)
+
+### Readings
```
Comment (ifcharming) [inline docs/cn/Contributing-to-Tachyon.md]: 延伸阅读？

**Comment by ifcharming** [comment]
Thanks for the translation! left a few minor comments.

**Comment by saltylin** [comment]
@ifcharming Well, I just thought these need not to be translated..

**Comment by ifcharming** [comment]
LGTM

Thanks @saltylin  -- it's not big deal :)

**Comment by yupeng9** [comment]
LGTM. Thanks for the translation.

**Pull request merged**